runtime/near-wallet-contract/wallet-contract/src/lib.rs

```diff
@@ -0,0 +1,53 @@
+//! Temporary implementation of the Wallet Contract.
```

Comment:
The fact that this implementation passes the integration test above makes me think: when making this production-ready, we should also have at least a test that has the right public key but an invalid signature

Reply:
Yes, we definitely must have. A plenty of new tests will be added when the real Wallet Contract implementation is ready. And there will be audit process.